applications/benchmark/BuildGradient.cpp

```diff
@@ -1,111 +1,15 @@
-#include "Benchmark.h"
-#include <Eigen/Core>
-#include "mechanics/elements/ElementOutputBlockVectorDouble.h"
-#include "mechanics/dofSubMatrixStorage/BlockFullVector.h"
-#include "mechanics/MechanicsEnums.h"
-#include "mechanics/elements/ElementShapeFunctions.h"
-#include "mechanics/constitutive/laws/EngineeringStressHelper.h"
-#include "mechanics/nodes/NodeDof.h"
+#include <benchmark/benchmark.h>
 #include "mechanics/nodes/NodeSimple.h"
-#include "mechanics/structures/unstructured/Structure.h"
-#include "mechanics/dofSubMatrixStorage/BlockFullMatrix.h"
 #include "mechanics/integrationtypes/IntegrationTypeTensorProduct.h"
-#include "mechanics/elements/ElementBase.h"
 #include "mechanics/elements/ElementFem.h"
 #include "mechanics/cell/Cell.h"
 #include "mechanics/interpolation/InterpolationQuadSerendipity.h"
 #include "mechanics/integrands/MomentumBalance.h"
-#include "mechanics/constitutive/laws/LinearElastic.h"
-#include "mechanics/sections/SectionPlane.h"
+#include "mechanics/constitutive/LinearElastic.h"
 
 namespace Benchmark
 {
```

Comment:
I find it quiet confusing to use the namespace "benchmark". Whenever I read a line starting with `Benchmark::` I think of a library feature until I see that you introduced this namespace. Maybe you should use a fixture here anyways. Then you have everything included into your benchmark and also don't need to write down the namespace.